### DISCUSSION TITLE
RAPv5/HRRRv4 Prototype access

### DISCUSSION THREAD

**Discussion by thefaj** [NOAA-GSL/GSL-Regional-Model-Forum#5] [General]
Dear NOAA,

Congratulations on the new discussion forum move! I like it!

I saw a message on the previous forum that mentioned you could grant access to the RAPv5/HRRRv4 GRIB2 grids on request: 
>The real-time experimental ESRL/GSD RAP/HRRR (RAPv5/HRRRv4 prototype) GRIB2 grids are available (password protected, access available upon request) at: 
>ftp://gsdftp.fsl.noaa.gov/rap (RAP) 
>ftp://gsdftp.fsl.noaa.gov/hrrr (CONUS HRRR) 
>ftp://gsdftp.fsl.noaa.gov/hrrr_ak/alaska (Alaska HRRR)
>
>Additional real-time experimental ESRL/GSD products: 
>ftp://gsdftp.fsl.noaa.gov/hrrr_hi/hawaii (Hawaii HRRR) 
>ftp://gsdftp.fsl.noaa.gov/rap_smoke (RAP wildfire smoke) 
>ftp://gsdftp.fsl.noaa.gov/hrrr_smoke/conus (CONUS HRRR wildfire smoke) 
>ftp://gsdftp.fsl.noaa.gov/hrrre/conus/ (CONUS HRRR ensemble) 
>ftp://gsdftp.fsl.noaa.gov/hrrr/conus/wrftwo_ens/ (CONUS HRRR time-lagged ensemble)
https://ruc.noaa.gov/forum/f2/Welcome.cgi/read/2599

If that's still available, may I request access?
Thank you,
-Alex Fajkowski

**Comment by EricJames-NOAA** [comment]
@thefaj, thanks for your interest in the RAPv5/HRRRv4 grids.  The post you refer to dates from before the operational implementation of RAPv5/HRRRv4, which occurred in Dec 2020.  So the grids are now available publicly from NCEP/NCO rather than from our GSL FTP server.  

RAPv5 grids are here: https://nomads.ncep.noaa.gov/pub/data/nccf/com/rap/prod/
And HRRRv4 grids are here: https://nomads.ncep.noaa.gov/pub/data/nccf/com/hrrr/prod/

We are still running the Hawaii and Caribbean domains of the HRRR locally, since those will not be implemented operationally, so we can provide access to our FTP site for those grids if you are interested in those domains.  

Thanks
Eric James

**Comment by thefaj** [comment]
Hi @EricJames-NOAA, Thank you for the quick response! That's exactly what I was looking for (and great data!). I'll get in touch when I'm up & running (and could use Hawaii & Caribbean domain data, too).

Thanks again!
-Alex

**Reply by thefaj** [reply]
Just a quick note of thanks for this forum and the services that NOAA provides. 

I (finally) launched my Air Quality Index app that shows Near Surface and Vertical Column smoke overlays for the continental US (pulling from HRRR).

If you’re interested, the app is free:
https://apps.apple.com/us/app/air-quality-index/id1296674639

Thank you!
-Alex

**Comment by nmoisseeva** [comment]
Hi @EricJames-NOAA,
I understand this is an old thread but looks like HRRR-HI grib data is still not publicly available. We are running operational air quality forecasts for HI State, and could benefit from the high-resolution HRRR offers. 
Is it possible to obtain FTP access?

**Reply by EricJames-NOAA** [reply]
@nmoisseeva sorry for the late response.  Yes, HRRR-HI GRIB2 data are still available on our FTP server.  If you can share your email address, I can reach out with access instructions.

**Comment by ViperG** [comment]
Hi @EricJames-NOAA,
Would I be able to request HRRR-HI GRIB2 access from your ftp server as well? We are running simulated fire propagations for risk analysis and could use the higher resolution wind data.
